index.html

```diff
@@ -10,39 +10,250 @@
+          </section>
+          <section class="work">
+            <img
+               width="350px"
+              src="./image/work4.png"
+            />
+            <h3 class="work-title">MUlti-Post Stories</h3>
+            <div class="profession">
+              <p>CANOPY</p>
+              <div class="expert">
+                <div class="bullet"></div>
+                <p>Back End Dev</p>
+              </div>
+              <div class="expert">
+                <div class="bullet"></div>
+                <p>2015</p>
+              </div>
+            </div>
+            <h3 class="description">
+              A daily selection of privately personalized reads;no accounts or
+              sign-ups required
+            </h3>
+    
+              <ul class="languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
+             
+              <button class="see-project">See project</button>
+              </section>
+              <section class="work">
+<h3 class="work-title">About my self</h3>
+<p id="about">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+<div class="social">
+  <h3 class="connect">LET'S CONNECT</h3>
+  <ul class="social-links">
+    <li>
+      <a
+        href="https://twitter.com/"
+        target="_blank"
+        rel="noopener noreferrer"
+        ><img src="./image/twitter.png" alt="twitter.icon"
+      /></a>
+    </li>
+    <li>
+      <a
+        href="https://www.linkedin.com/in/abenezer-samuel/"
+        target="_blank"
+        rel="noopener noreferrer"
+        ><img src="./image/linkdin.png" alt="linkdin logo"
+      /></a>
+    </li>
+
+    <li>
+      <a href="" target="_blank" rel="noopener noreferrer"
+        ><img src="./image/medium.png" alt="medium logo"
+      /></a>
+    </li>
+    <li>
+      <a
+        href="https://github.com/abenisamuel11"
+        target="_blank"
+        rel="noopener noreferrer"
+        ><img src="./image/Github.png" alt="github logo"
+      /></a>
+    </li>
+
+    <li>
+      <a href="" target="_blank" rel="noopener noreferrer"
+        ><img src="./image/A.png" alt="angle list icon"
+      /></a>
+    </li>
+    </ul>
+    <button class="Get-my-resume">Get my resume</button> 
+    <ul class="lan-tools">
+    <div class="java-icon"> 
+      <li><img src="./image/javascript.png" alt="javascript logo"></li><br>
+      <p>javascript</p>
+    </div>
+    <div class="html-icon">
+      <li><img src="./image/html.png" alt="html logo"></li><br>
+       <p>html</p>
+    </div>
+    <div class="css-icon">
+      <li><img src="./image/css.png" alt="css logo"></li><br>
+<p>css</p>
+</div>
+</ul>
+<p id="frame">frameworks</p><hr color="silver" size="1px" width="80%">
+<p id="skills">skills</p>
```

Comment:
- [ ] Please lets try as much as possible to ensure that the professional skills subsection matches the Figma template design. At this level, the languages should be aligned column with language icon and name aligned in row format. The underlines on frameworks and skills should also be fixed to match the Figma design. In addition ensure that the `down-arrow` and `right-arrows` are also added with the language title. This is to ensure that our portifolio is more user friendly ☺️.
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/236162814-27c89185-9f18-490c-b70e-4c897105bfe5.png)

Figma template design 😎 
![image](https://user-images.githubusercontent.com/99394374/236162974-aaf04516-fbc6-4296-9cc0-e1ba52e20bad.png)